src/core.ts

```diff
@@ -9,8 +9,24 @@ export const apply = (
 ): StyleProp<Styles | {}> =>
   styles.flatMap(s => (typeof s === 'string' ? C[s] : s));
 
-export const classNames = (classes: string) =>
-  classes.split(' ').map(name => C[name]);
+export const classNames = (
+  classes: string,
+  conditionalClasses?: DynamicObject<boolean>
+) => {
+  const conditionals: string =
+    typeof conditionalClasses !== 'undefined'
+      ? Object.entries(conditionalClasses || {}).reduce(
+          (current, [key, value]) =>
+            value ? `${current} ${key}`.trim() : current.trim(),
+          ''
+        )
+      : '';
+
+  return `${classes} ${conditionals}`
+    .trim()
+    .split(' ')
+    .map(name => C[name]);
+};
```

Comment:
Gracias por el PR eze !
Te hago un propuesta de hacer esta funcion un poco mas funcional, que te parece ?

```suggestion
export const classNames = (classes: string, conditionalClasses: DynamicObject<boolean> = {}) =>
  Object.entries(conditionalClasses)
    .reduce(
      (current, [key, value]) => (value ? current.concat(key) : current),
      classes.trim().split(' '),
    )
    .map(name => C[name]);
```

Reply:
Me gusta, lo cambio. Agrego un `.trim()` y `.split(' ')` a `key` ya que si no, no acepta multiples clases en la misma condición y genera un `undefined` si tiene mas de un espacio entre clase y clase